packages/components/src/Spinner/Spinner.css

```diff
@@ -0,0 +1,86 @@
+.spinner {
+  position: relative;
+  pointer-events: none;
+  /* 1 */
+  opacity: 0.6;
+}
+
+.spinner:after {
+  content: "";
+  display: block;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  z-index: 3;
+  width: 2.25rem;
+  height: 2.25rem;
+  -webkit-box-shadow: 0px 0px 0.125rem 0.1875rem rgba(1, 27, 37, 0.3);
```

Comment:
We have Atlantis will automatically generate any necessary browser specific rules like this.

Reply:
Totally! I removed the other webkit stuff too, didn't seem to be doing anything either! 😄